paella-matterhorn/javascript/05_initdelegate.js

```diff
@@ -58,22 +55,25 @@ function initPaellaMatterhorn(episodeId, onSuccess, onError) {
 					       paella.matterhorn.resourceId = "";
 					   }
 					   // end #DCE logging helper
-						
-						if (serie != undefined) {
-							asyncLoader.addCallback(new paella.JSONCallback({url:'/series/'+serie+'.json'}), "serie");
-							asyncLoader.addCallback(new paella.JSONCallback({url:'/series/'+serie+'/acl.json'}), "acl");
-						
-							asyncLoader.load(function() {
-									//Check for series
-									paella.matterhorn.serie = asyncLoader.getCallback("serie").data;
-									//Check for acl
-									paella.matterhorn.acl = asyncLoader.getCallback("acl").data;
-									if (onSuccess) onSuccess();
-								},
-								function() {
-									if (onError) onError();
-								}
-							);
+					   // start #DCE get series from serch endpoint (instead of series endpoint)
+					   if (serie != undefined) {
+					       base.ajax.get({
```

Comment:
No, I mean you could just put it in a function – not a new plugin – then call it here, like `loadSeries();`.